R/app-driver.R

```diff
@@ -140,12 +140,25 @@ NULL
 #'   [`chromote::ChromoteSession`]'s `$get_screenshot()` method. If missing, the
 #'   value will default to `$new(screenshot_args=)`.
 #'
-#' If the value is:
+#' If `screenshot_args` is:
 #'   * `TRUE`: A screenshot of the whole page will be taken with no delay
```

Comment:
When you say "whole page", does this mean the `"viewport"`, `"scroll"`, or something else?